setup.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import setuptools
```

Comment:
Why do we need setup.py to support 3.7?

Reply:
I have been publishing using setup.py on Pypi, but as discussed I will use pyproject.toml, and will create a separate PR for it. So, removed it.